convene-web/app/controllers/furniture_placements_controller.rb

```diff
@@ -0,0 +1,21 @@
+class FurniturePlacementsController < ApplicationController
+  def update
+    furniture_placement.update(furniture_placement_params)
```

Comment:
It's probably ok because we control the params in the markup but should be always do `update!`?